app/services/organizations/create_service.rb

```diff
@@ -51,6 +56,17 @@ def create_organization(name, slug, email)
     )
   end
 
+  def create_location(country, city_town, province_state)
+    ActsAsTenant.with_tenant(@organization) do
+      Location.create!(
+        organization: @organization,
```

Comment:
Do you need this line if we are scoping this within `ActsAsTenant.with_tenant(@organization) do`

Reply:
You are right, that line of code was unneeded. Wasn't aware of the ins-and-outs of the act_as_tenant gem. Pushed up a commit that removes the line.